src/robot.ts

```diff
@@ -6,7 +6,17 @@ import { isString } from 'util'
 
 export = (robot: Application) => {
   robot.on(['issues.opened', 'issues.reopened'], async context => {
-    const remoteConfig = await getContent(context, '/.github/issue-close-app.yml')
+    let remoteConfig: string | null = null
+    try {
+      remoteConfig = await getContent(context, '/.github/issue-close-app.yml')
+    } catch (e) {
+      if (e.hasOwnProperty('code') && e.code === 404) {
```

Comment:
comment wanted

Reply:
Why?  
Comments like "Config file not found" may be annoying I think. Why not just ignore this error?